shared/team-building/user-result.native.tsx

```diff
@@ -66,6 +66,8 @@ const Avatar = ({
 }) => {
   if (keybaseUsername) {
     return <Kb.Avatar size={AvatarSize} username={keybaseUsername} />
+  } else if (resultForService === 'keybase' || resultForService === 'contact') {
+    return <Kb.Avatar size={AvatarSize} username={'invalid username for placeholder avatar'} />
```

Comment:
we need some story for this because I ran to this same issue in my other PR for SBS profiles

Reply:
Sort of...this current approach does work correctly.